src/components/ClearingsUploader/ClearingsUploader.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Select from '../Select/Select';
+import SelectItem from '../Select/SelectItem';
+import FileUpload from '../FileUpload/FileUpload';
+import newTheme from '../../themes/new';
+
+const EMPTY = 'EMPTY';
+const UPLOADING = 'UPLOADING';
+const COMPLETED = 'COMPLETED';
+const UPLOAD_ERROR = 'UPLOAD_ERROR';
+const FILE_TYPE_ERROR = 'FILE_TYPE_ERROR';
+
+const propTypes = {
+  text: PropTypes.object,
```

Comment:
`text` sound like it's a single string.
```suggestion
  messages: PropTypes.shape({
    choose_processor: PropTypes.string,
    upload_your_file: PropTypes.string,
    drop_file_here: PropTypes.string,
    or_click_here_to_manually_upload: PropTypes.string,
    know_allowed_formats: PropTypes.string
  }),
```
And then you expect a flat object with keys => strings like:
```js
{
   choose_processor: 'Elegí la procesadora',
   upload_your_file: 'Subí tu archivo'
   drop_file_here: 'Arrastrá acá el archivo',
   or_click_here_to_manually_upload: 'o hace click acá y subilo manualmente',
   know_allowed_formats: 'Conocé los formatos permitidos'
}
```
You need to also pass messages to the FileUpload descendant and use them there.
If you need more, just add them here, this is what provides the best developer experience IMO